listenbrainz/webserver/views/index.py

```diff
@@ -120,6 +124,35 @@ def current_status():
     )
 
 
+@index_bp.route('/delete-user/<musicbrainz_id>')
+def mb_user_deleter(musicbrainz_id):
+    _authorize_mb_user_deleter(request.args.get('auth_token', ''))
+    delete_user(musicbrainz_id)
+    return jsonify({'status': 'ok'}), 200
+
+
+def _authorize_mb_user_deleter(auth_token):
+    query_url = 'https://musicbrainz.org/oauth2/userinfo'
+    headers = {'Authorization': 'Bearer {}'.format(auth_token)}
+    r = requests.get(query_url, headers=headers)
+    try:
+        r.raise_for_status()
+    except HTTPError:
+        raise Unauthorized('Not authorized to use this view')
```

Comment:
I like how you give no useful info the caller. Well done!

Reply:
https://httpstatuses.com/401 - `Unauthorised` requires a "WWW-Authenticate" header. Does this get set internally?

Reply:
Doesn't seem like werkzeug sets it (https://github.com/pallets/werkzeug/blob/master/werkzeug/exceptions.py#L214)